system/CodeIgniter.php

```diff
@@ -286,7 +286,7 @@ protected function initializeKint()
      * @throws Exception
      * @throws RedirectException
      *
-     * @return bool|mixed|RequestInterface|ResponseInterface
+     * @return bool|mixed|RequestInterface|ResponseInterface|void
```

Comment:
Just to comment (not related to this PR), this is a sign of bad design. This could be reduced to `mixed|void`, but then again a method that may or may not return anything is nothing but bad.

Reply:
We have a lot of this. 🤦‍♂️ 
My hope is that in version with a move to typing everything (and maybe even strict mode?) a lot of this will be resolved by necessity.

Reply:
I don't like `mixed`. I want to remove it if possible.